polaris-react/src/components/Checkbox/Checkbox.scss

```diff
@@ -158,48 +124,34 @@
 
 .Icon {
   position: absolute;
-  top: 50%;
-  left: 50%;
   transform-origin: 50% 50%;
   pointer-events: none;
-
-  transform: translate3d(-50%, -50%, 0) scale(0.25);
   opacity: 0;
-  transition: opacity var(--p-motion-duration-100) var(--p-motion-ease),
-    transform var(--p-motion-duration-100) var(--p-motion-ease);
-
-  #{$se23} & {
-    top: calc(-1 * var(--p-space-05));
-    left: calc(-1 * var(--p-space-05));
-    bottom: calc(-1 * var(--p-space-05));
-    right: calc(-1 * var(--p-space-05));
-    /* stylelint-disable-next-line  -- se23 override */
-    transform: none !important;
```

Comment:
🤔 Hmmm, the `!mportant` here implies there was another style rule overriding the `transform: none` even though the `#{$se23} & {` selector was more specific.

By removing it now, there may still be a `transform` being applied from somewhere that we don't want... Can you confirm that's no longer the case?